### DISCUSSION TITLE
CSS Classes instead of Hex Color codes?

### DISCUSSION THREAD

**Discussion by BrekiTomasson** [spatie/shiki-php#6] [Q&A] [answered]
Sorry if this question is answered elsewhere or if it makes perfect sense when using the package; I just had a thought when reading the blog post on `freek.dev` that I haven't seen answered elsewhere. When using this package, the HTML that is generated contains things like `<pre class="shiki" style="background-color: #2e3440ff">`. 

Would it be possible, instead of using the `style` tag, for every different type of tag to be tranlated into a `CSS` class instead? I feel it would make it far more convenient to modify the presentation of the output that way, and it would allow people to generate their own "themes" as well.

**Comment by riasvdv** [comment, marked as answer]
This isn't possible, what you can do is adapt a VS Code theme to your liking and pass that to Shiki instead.

**Reply by BrekiTomasson** [reply]
I assumed as much, just wanted to confirm. Thanks!
